packages/fxa-auth-client/test/crypto.ts

```diff
@@ -1,6 +1,10 @@
-import assert from 'assert';
+import * as assert from 'assert';
 import '../server'; // must import this to run with nodejs
-import { getCredentials, unbundleKeyFetchResponse } from 'fxa-auth-client/lib/crypto';
+import { getCredentials, hkdf, jweDecrypt, jweEncrypt, unbundleKeyFetchResponse } from 'fxa-auth-client/lib/crypto';
+import { randomKey, getRecoveryKeyIdByUid } from 'fxa-auth-client/lib/recoveryKey';
+import { hexToUint8 } from '../lib/utils';
```

Comment:
Not sure if it matters, but some of these use `../` and others use `fxa-auth-client/` (even though this is the same package).